sdk/src/types/block/payload/transaction/essence/regular.rs

```diff
@@ -208,9 +234,14 @@ impl RegularTransactionEssence {
         self.network_id
     }
 
-    /// Returns the creation time of a [`RegularTransactionEssence`].
-    pub fn creation_time(&self) -> u64 {
-        self.creation_time
+    /// Returns the slot index of the block in which the transaction was created. [`RegularTransactionEssence`].
```

Comment:
Is this supposed to be?
```suggestion
    /// Returns the slot index of the block in which the [`RegularTransactionEssence`] was created.
```